scripts/channel_map.py

```diff
@@ -3,8 +3,8 @@
 class ChannelMap():
     channel_map = {
         'main': {
-            'tfm': 'net8.0',
-            'branch': '8.0',
+            'tfm': 'net9.0',
+            'branch': '9.0',
             'quality': 'daily'
         },
         '8.0': {
```

Comment:
We should add 9.0 also to the channel_map, though that can be done in another PR.

Reply:
done.